app/helpers/alchemy/admin/essences_helper.rb

```diff
@@ -80,6 +80,25 @@ def essence_picture_thumbnail(content, options)
         )
       end
 
+      # Show image cropping link for given content and options?
+      def allow_image_cropping?(content, options = {})
```

Comment:
What do you think about implementing this method to the related `essence` instead of having another helper method?